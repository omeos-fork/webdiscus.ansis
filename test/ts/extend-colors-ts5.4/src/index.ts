import ansis from 'ansis';

const myTheme = {
  pink: '#FF75D1',
  orange: '#FFAB40',
  apple: '#4FA83D',
};

ansis.extend(myTheme);

console.log(ansis.pink.underline('Hello'));