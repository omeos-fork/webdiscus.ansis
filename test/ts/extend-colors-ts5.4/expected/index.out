[38;2;255;117;209m[4mHello[24m[39m
